general/solomiia_tymoshchuk/Task1.java

```diff
@@ -0,0 +1,53 @@
+package general.solomiia_tymoshchuk;
+
+import java.util.Scanner;
+
+/**
+ * Write a program that accepts numbers of miles as a floating point number,
+ * converts it to kilometers and outputs the result.
+ **/
+
+public class Task1 {
+    double kms;
```

Comment:
Why do we need these instance variables if we pass same values directly to the methods?